onnxruntime/test/mlas/unittest/test_fp16_activation.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "test_fp16.h"
+
+#ifdef MLAS_F16VEC_INTRINSICS_SUPPORTED
+
+class MlasFp16ActivationTest : public MlasTestBase {
+ public:
+  static const char* GetTestSuiteName() {
+    static const std::string suite_name("Fp16Activation");
+    return suite_name.c_str();
+  }
+
+  void ExecuteShort(void) override {
+    union AliasedValue {
+      unsigned u;
+      float f;
+    };
+
+    // N.B. The test data includes values at the edge of Tanh/Logistic boundaries.
+    //    Identity,     Relu,         LeakyRelu,    Tanh,         Logistic,     Clip,
+    static const AliasedValue TestData[] = {
```

Comment:
Seems it lakes test data for Relu, LeakyRelu, Clip and MlasHardSigmoidActivation? I assumed it should be `AliasedValue TestData[20][7]` like test_activation.cpp. By the way, test_activation.cpp seems lakes test data for MlasHardSigmoidActivation and it doesn't test it.

Reply:
it reuses the same data and test each of the supported activation kind. The result is compared against fp32 implementation.